requirements.txt

```diff
@@ -1,14 +1,15 @@
 # These are only a subset of all requirements - the base requirements are also
 # defined in the ./hs_docker_base/Dockerfile file.
 
-djangorestframework==3.6.4
-robot_detection==0.4
 django-ipware==2.1.0
-django-test-without-migrations==0.6
 django-rest-swagger==2.2.0
+django-sass-processor==0.7
+django-scribbler==0.9.0
+django-test-without-migrations==0.6
+djangorestframework==3.6.4
+factory-boy==2.8.1
 jsonschema==2.6.0
+libsass==0.14.5
 nameparser==0.5.6
 python-dotenv==0.8.2
-libsass==0.14.5
-django-sass-processor==0.7
-factory-boy==2.8.1
+robot_detection==0.4
```

Comment:
Since we're in here touching these, lets remove the 'definitely not myhpom' holdovers from hydroshare: robot_detection, nameparser, jsonschema, djangorestframework, django-test-without-migrations, django-rest-swagger. (when I'd pulled these into this file, we hadn't decided to disable hydroshare at that point - these probably should have been removed when we did that).